docs/ai/text/generation.md

```diff
@@ -87,12 +87,15 @@ Marvin can generate synthetic data according to a schema and instructions. Gener
         !!! success "Result"
             
             ```python
-            assert locations == [
+            print(locations)
```

Comment:
This is tough, assert won't work for all users but the comment doesn't show syntax highlighting and is harder to read. Happy to merge this now and will revisit

Reply:
updated so we have syntax highlighting